src/app/view/perspectives/post-it/document/post-it-document.component.html

```diff
@@ -1,7 +1,10 @@
 <div class="card-header d-inline-flex py-1 px-3">
   <i class="icon fa-fw {{ postItModel.document.collection.icon }} mr-1"
-     [style.color]="postItModel.document.collection.color"></i>
-  <div class="collection-name" [style.color]="postItModel.document.collection.color">
+     [style.color]="postItModel.document.collection.color"
+     title="{{ postItModel.document.collection.name }}"></i>
+  <div class="collection-name"
+       [style.color]="postItModel.document.collection.color"
+       title="{{ postItModel.document.collection.name }}">
```

Comment:
Why is the color as well as title defined on both elements? Would not it work if a new parent element was created for those two and it was defined there just once?